### ISSUE TITLE
Geometry: Point List Normal

### ISSUE THREAD

**Issue by RodeyOrg** [JacquesLucke/animation_nodes_manual#53]
Hello,

Wondering could someone please outline a little more on the example used here, especially in how the example is setup (for instance the empties as highlighted). I got quite far but not close enough. Still grasping Point List Normal concept but fleshing out the example may help. 

[Points List Normal Example](https://animation-nodes-manual.readthedocs.io/en/latest/user_guide/nodes/geometry/point_list_normal.html#examples-of-usage)

My Example

![screen shot 2018-11-02 at 23 36 27](https://user-images.githubusercontent.com/44178760/47945232-7de1b600-def8-11e8-9c62-562b8b08004c.png)

Not sure how key the geometry nodes will be in everyday motion graphics use but would be great to get it.

**Comment by OmarSquircleArt** [comment]
When we made that example, we hooked each vertex of a plane with one of the empties using four hook modifiers. That's why when you move the empties, the plane move with it, however, this is not the point of the example, we did that to visualize the plane defined by the four points so that the reader understands the relation between the orientation of the cube and the normal of the plane.

But you are right, it is confusing, we should make a better example.
